orchagent/bufferorch.cpp

```diff
@@ -11,8 +11,6 @@ extern sai_queue_api_t *sai_queue_api;
 extern sai_switch_api_t *sai_switch_api;
 extern sai_buffer_api_t *sai_buffer_api;
 
-extern PortsOrch *gPortsOrch;
-
```

Comment:
Completely agree, but... if I do this in current PR - it will result the same because PRs are usually merged with "squash" option. And here is never right time for "general code improvement" task :(